src/PsTool/Common.cs

```diff
@@ -3038,8 +3038,10 @@ public static string GetLeadingType(Dictionary<string, string> Properties)
 		public static string GetSaveInFolder(string template, string database, string layout)
 		{
 			Dictionary<string, string> map = new Dictionary<string, string>();
-
-			map["Documents"] = ManageDirectory.ShortFileName(Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments));
+			if(UsingMonoVM)

```

Comment:
1. I am not sure why this should be a runtime test and not a compile time test?

2. ShortFileName has code for Linux. Shouldn't that code be corrected instead of avoiding it?